README.md

```diff
@@ -1,33 +1,60 @@
-# bioio
+# BioIO
 
 [![Build Status](https://github.com/bioio-devs/bioio/actions/workflows/ci.yml/badge.svg)](https://github.com/bioio-devs/bioio/actions)
 [![Documentation](https://github.com/bioio-devs/bioio/actions/workflows/docs.yml/badge.svg)](https://bioio-devs.github.io/bioio)
 [![PyPI version](https://badge.fury.io/py/bioio.svg)](https://badge.fury.io/py/bioio)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![Python 3.6](https://img.shields.io/badge/python-3.9,3.10,3.11-blue.svg)](https://www.python.org/downloads/release/python-360/)
 
-
-Image reading, metadata management, and image writing for Microscopy images in Python
+Image Reading, Metadata Conversion, and Image Writing for Microscopy Images in Pure Python
 
 ---
 
-## Installation
-
-**Stable Release:** `pip install bioio`<br>
-**Development Head:** `pip install git+https://github.com/bioio-devs/bioio.git`
+## Documentation
 
-## Quickstart
+[See the full documentation on our GitHub pages site](https://bioio-devs.github.io/bioio/OVERVIEW.html)
 
-```python
-from bioio import example
+## Example Usage (see full documentation for more examples)
```

Comment:
"see full documentation for more examples" is repeated here and also just below. I like the linked version of it :)

Nit/optional: Maybe this section could be called "Quick Start" instead of "Example Usage"

Reply:
I named it "Example Usage" because our "Quick Start" in the full docs is a different set of content. If we change this to "Quick Start" it seems the docs should change to something else